app/components/Nav/Mobile/navMobile.css

```diff
@@ -0,0 +1,59 @@
+@media only screen and (max-width: 640px) {
+  .top-header {
+    padding: 0;
+  }
+  .top-nav {
+    justify-content: space-between;
+    padding: var(--spacing-20) var(--spacing-24);
```

Comment:
The nav was supposed to be based on height (using 80px) rather than padding. If you want to use padding can you just round to the nearest one, so either 16px or 24px? And remove the --spacing-20 var